### PR TITLE
[Form] remove deprecated getTimezones() method

### PR DESCRIPTION
**Pull request by xabbuh**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets |  |
| License | MIT |
| Doc PR |  |

### CODE REVIEW

File: src/Symfony/Component/Form/Extension/Core/Type/TimezoneType.php [hunk 1]
```diff
@@ -66,46 +59,6 @@ public function getBlockPrefix()
...
-     * Returns the timezone choices.
-     *
-     * The choices are generated from the ICU function
-     * \DateTimeZone::listIdentifiers(). They are cached during a single request,
-     * so multiple timezone fields on the same page don't lead to unnecessary
-     * overhead.
-     *
-     * @return array The timezone choices
      */
     private static function getFlippedTimezones()
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Form/Extension/Core/Type/TimezoneType.php]: what about renaming this to getTimezones now?

Comment (Tobion) [inline src/Symfony/Component/Form/Extension/Core/Type/TimezoneType.php]: agree

Comment (xabbuh) [inline src/Symfony/Component/Form/Extension/Core/Type/TimezoneType.php]: done

**Comment by fabpot** [comment]
Thank you @xabbuh.

**Pull request merged**